src/test/scala/edu/ie3/simona/model/participant/WecModelSpec.scala

```diff
@@ -96,7 +98,7 @@ class WecModelSpec extends UnitSpec with DefaultTestData {
       val wecModel = buildWecModel()
       wecModel.uuid shouldBe inputModel.getUuid
       wecModel.id shouldBe inputModel.getId
-      wecModel.rotorArea shouldBe typeInput.getRotorArea
+      wecModel.rotorArea shouldBe (typeInput.getRotorArea === rotorAreaTolerance)
```

Comment:
Since left side (wecModel.rotorArea) comes as Squants SquareMeter and typeInput.getRotorArea comes as ComparableQuantity[SquareMeter] the best way will be to convert to double, i guess.


```
wecModel.rotorArea.toSquareMeters shouldBe approximate (typeInput.getRotorArea.toSystemUnit.getValue.doubleValue())
```